tools/model_converter/model_converter.py

```diff
@@ -0,0 +1,591 @@
+#!/usr/bin/env python3
```

Comment:
maybe replace with `#!/usr/bin/env -S uv run --script` and make this file executable?